src/test/java/com/github/stefanbirkner/systemlambda/RestoreSystemErrChecks.java

```diff
@@ -7,7 +7,7 @@
 import static com.github.stefanbirkner.fishbowl.Fishbowl.ignoreException;
 import static org.assertj.core.api.Assertions.assertThat;
 
-class RestoreSystemErrChecks {
+abstract class RestoreSystemErrChecks {
```

Comment:
This is just a test base class and JUnit5 cannot run it. If marked `abstract` the IDE won't try to run it either. Obviously, its name prevents it being run by surefire, but this covers both angles.